packages/caver-rpc/src/governance.js

```diff
@@ -161,13 +161,37 @@ const Governance = function Governance(...args) {
          * const result = await caver.rpc.governance.getChainConfig()
          *
          * @param {function} [callback] Optional callback, returns an error object as the first parameter and the result as the second.
-         * @return {Promise<object>} The initial chain configuration
+         * @return {Promise<object>} The latest chain configuration
          */
         new Method({
             name: 'getChainConfig',
             call: 'governance_chainConfig',
             params: 0,
         }),
+        /**
+         * Returns the chain configuration at specific block.
+         * If the parameter is not set, it returns the chain configuration at the latest block.
+         * To see the current information, please use {@link itemsAt}.
+         *
+         * @memberof Governance
+         * @method getChainConfigAt
+         * @instance
+         *
+         * @example
+         * // Use `caver.rpc.governance.getChainConfigAt()`
+         * const result = await caver.rpc.governance.getChainConfigAt(0)
+         * const result = await caver.rpc.governance.getChainConfigAt('0x58482921af951cf42a069436ac9338de50fd963bdbea40e396f416f9ac96a08b')
```

Comment:
Klaytn node doesn't support the blockHash parameter.
Please check https://github.com/klaytn/klaytn/blob/dev/governance/api.go#L305.

Reply:
Furthermore, please check other APIs you added or changed.

Reply:
I checked other APIs I added or changed. 
Thank you.

Reply:
`const result = await caver.rpc.governance.getChainConfigAt('0x58482921af951cf42a069436ac9338de50fd963bdbea40e396f416f9ac96a08b')` need to remove example code also

Reply:
I pushed the commit.
Thank you for your detailed review.